test/scripts/lib.sh

```diff
@@ -1,17 +1,18 @@
 #!/bin/bash
 
 retry() {
+  local command=$@
   local n=1
   local max=5
-  local delay=5
+  local delay=10
```

Comment:
@jewertow Not sure that is significant change. I noticed that more often than not I have two failed attempts for installing MetalLB, so I bumped retry to 10s based on scientific measures. However, new evidence suggests `8s` would be more accurate to avoid quantum entanglement.